app/code/Magento/SendFriend/Helper/Data.php

```diff
@@ -49,6 +53,36 @@ public function isEnabled($store = null)
         );
     }
 
+    /**
+     * Check if email should be sent from store
+     *
+     * @param int $store
+     * @return bool
+     */
+    public function isSentFromStore($store = null)
+    {
+        return $this->scopeConfig->isSetFlag(
+            self::SENT_FROM_STORE,
+            \Magento\Store\Model\ScopeInterface::SCOPE_STORE,
+            $store
+        );
+    }
+    
+    /**
+     * Get the email address of the store
+     *
+     * @param int $store
+     * @return string
+     */
+    public function getStoreEmail($store = null)
```

Comment:
Unfortunately, we are not able to add new public methods to the @api classes according to our [Backward Compatible Development Guide](https://devdocs.magento.com/guides/v2.3/contributor-guide/backward-compatible-development/)